components/scream/data/scream_input.yaml

```diff
@@ -18,4 +18,113 @@ Atmosphere Driver:
     Reference Grid: Physics GLL
     Dynamics Driven:
       Dynamics Namelist File Name: ./data/theta.nl
+
+SCREAM:
+
+  Start Date: 01-01-2000 # For CIME cases, do NOT set this! It will be set by CIME's RUN_STARTDATE
+
+  # Model Timesteps
+  whole_model_dt: 3600.0 #aka phys/dyn coupling dt
```

Comment:
I appreciate this comment.  It was super confusing to me what timesteps meant way back when I started running E3SM.